File: VB/DXSample/ViewModel/NextDetailViewModel.vb
Imports DevExpress.Mvvm

Namespace DXSample.ViewModel

    Public Class NextDetailViewModel

        Public Overridable ReadOnly Property NavigationService As INavigationService
            Get
                Return Nothing
            End Get
        End Property

        Public Sub NavigateHome()
            NavigationService.Navigate("HomeView")
        End Sub
    End Class
End Namespace

File: CS/DXSample/ViewModel/NextDetailViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;

namespace DXSample.ViewModel {
    public class NextDetailViewModel {
        public virtual INavigationService NavigationService { get { return null; } }
        public void NavigateHome() {
            NavigationService.Navigate("HomeView");
        }
    }
}
